.ci/scripts/test.sh

```diff
@@ -1,19 +1,15 @@
-#!/usr/bin/env bash
-set -euxo pipefail
+#!/bin/sh
+
+set -e
 
-GO111MODULE=off go get -u github.com/elastic/go-licenser
 go mod verify
-go-licenser -d
-go run .ci/scripts/check_format.go
-go run .ci/scripts/check_lint.go
+go run github.com/elastic/go-licenser@latest -d
+sh .ci/scripts/format.sh
```

Comment:
Are these extra scripts needed? It feels unbalanced to have the .bat call the commands directly and to have this invoke a separate script. The scripts seem simple enough to call the commands directly.

Reply:
Agree, moved to the same file :+1: